packages/rlc-codegen/src/interfaces.ts

```diff
@@ -35,3 +37,14 @@ export interface OperationAnnotations {
   isLongRunning?: boolean;
   isPageable?: boolean;
 }
+
+export interface RLCOptions {
+  includeShortcuts?: boolean;
```

Comment:
Currently cadl doesn't support we input command line agruments, I guess this is potential blocker for our August MVP ?

Reply:
I think the [sidecar ](https://github.com/microsoft/cadl/issues/218) feature could be used to comsume metadata for versining, any customizations as well as other configurations. 

Maybe the command line arguments could be fed into `typescript.cadl` file as input similar to the `readme.md` file.

@joheredi  How do you think?

Reply:
Yeah it looks like sidecar will help here! TBH I hadn't looked into how to get those options :P I just assumed there will be a way to do so.

@qiaozha I don't think it will be a blocker, worst case scenario we set it to `false`, shortcuts are not on by default yet so no harm done for MVP

Reply:
Yeah, I haven't saw any options to set them either, just assume finally we would have a way to do so.

If there is no way to set arguments now maybe we just set default values for them